src/api/network_test.go

```diff
@@ -102,13 +131,40 @@ func TestConnection(t *testing.T) {
 }
 
 func TestConnections(t *testing.T) {
+	bp := daemon.BlockchainProgress{
+		Current: 35,
+		Highest: 39,
+		Peers:   nil}
+	bp.Peers = append(bp.Peers, struct {
+		Address string `json:"address"`
+		Height  uint64 `json:"height"`
+	}{
+		Address: "127.3.5.1",
+		Height:  39,
+	})
+	bp.Peers = append(bp.Peers, struct {
+		Address string `json:"address"`
+		Height  uint64 `json:"height"`
+	}{
+		Address: "127.0.0.1",
+		Height:  12,
+	})
+	bp.Peers = append(bp.Peers, struct {
+		Address string `json:"address"`
+		Height  uint64 `json:"height"`
+	}{
+		Address: "127.0.5.1",
+		Height:  13,
+	})
+
 	tt := []struct {
-		name                        string
-		method                      string
-		status                      int
-		err                         string
-		gatewayGetConnectionsResult *daemon.Connections
-		result                      *daemon.Connections
+		name                               string
+		method                             string
+		status                             int
+		err                                string
+		gatewayGetConnectionsResult        *daemon.Connections
+		gatewayGetBlockchainProgressResult *daemon.BlockchainProgress
```

Comment:
should have test that mocks GetBlockchainProgress with an error

Reply:
done